### PR TITLE
adjust upgrade guide about obsoleted throttling API

### PR DESCRIPTION
**Pull request by timbussmann**
related to https://github.com/Particular/NServiceBus/issues/2859

### CODE REVIEW

**Comment by timbussmann** [comment]
@Particular/documentation please review

File: nservicebus/upgrades/5to6.md [hunk 1]
```diff
@@ -301,7 +301,9 @@ snippet:5to6ScanningUpgrade
 
 ## Throttling
 
-Throttling of message processing can no longer be adjusted at run time and will as of Version 6 require the endpoint to be restarted to take effect. You can read more about throttling [here](/nservicebus/operations/tuning.md).
+Requirements to throttling mechanisms are very different. While some 3rd party services (e.g. GitHub, Twitter, Google, etc.) enforce rate limits on certain time periods, other services may have entirely different usage limitations. The previous throttling API offered a very limited, messages per second based, throttling mechanism which only works for very few scenarios. Therefore we decided to remove the throttling API with Version 6 without a built-in alternative. Instead we provide you with the tools to implement your own throttling mechanism which fits your needs. Visit the [tuning NServiceBus documentation](/nservicebus/operations/tuning.md) to learn more about the available options.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: remove "we" "your" etc

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: i think `[tuning NServiceBus]` is enough

File: nservicebus/upgrades/5to6.md [hunk 2]
```diff
@@ -301,7 +301,9 @@ snippet:5to6ScanningUpgrade
...
+
+The `MaximumMessageThroughputPerSecond` on `TransportConfig` has been marked as obsolete. If you're using a configuration based approach, the endpoint will fail to start when using the `MaximumMessageThroughputPerSecond` attribute on the `<TransportConfig>` element.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: > If you're using a configuration based

 isnt this implied by 

> using the `MaximumMessageThroughputPerSecond` attribute on the `<TransportConfig>` element.

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: remove `you'r`

**Comment by timbussmann** [comment]
@Particular/documentation updated. Please re-review.

File: nservicebus/upgrades/5to6.md [hunk 3]
```diff
@@ -301,12 +301,14 @@ snippet:5to6ScanningUpgrade
...
+
+The `MaximumMessageThroughputPerSecond` on the `TransportConfig` class has been marked as obsolete. Using a configuration based approach, the endpoint will fail to start when using the `MaximumMessageThroughputPerSecond` attribute on the `<TransportConfig>` element.
 
 
 ## RepeatedFailuresOverTimeCircuitBreaker has been made internal
 
-If your are using it just include [the source code](https://github.com/Particular/NServiceBus/blob/5.2.5/src/NServiceBus.Core/CircuitBreakers/RepeatedFailuresOverTimeCircuitBreaker.cs) in your project.
+If you are using it just include [the source code](https://github.com/Particular/NServiceBus/blob/5.2.5/src/NServiceBus.Core/CircuitBreakers/RepeatedFailuresOverTimeCircuitBreaker.cs) in your project.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: remove the "you"

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: can we have better link text than "the source code"

Comment (timbussmann) [inline nservicebus/upgrades/5to6.md]: @SimonCropp that chapter is actually not from my changes. I just fixed the "your are", but I'm happy to fix it. How about:
*We recommend to use a independent Circuit Breaker implementation. Please feel free to copy [this implementation].*

**Pull request merged**